Neos.Neos/Resources/Private/Templates/Module/Management/History/Index.html

```diff
@@ -45,38 +45,40 @@
 </f:if>
 
 <script>
-	$(function() {
+	window.addEventListener('DOMContentLoaded', (event) => {
+		jQuery(function() {
 
-		var nextPage = '{nextPage}';
-		var historyContainer = $('.neos-history');
-		$('.loadMore button').click(function() {
+			var nextPage = '{nextPage}';
+			var historyContainer = $('.neos-history');
+			jQuery('.loadMore button').click(function() {
```

Comment:
Yeah in general I don't think we should replace all the dollars without good reason.